tests/php/test-class-customize-snapshot-manager.php

```diff
@@ -729,4 +729,40 @@ public function test_replace_customize_link() {
 		$parsed_preview_url = wp_parse_url( $query_params['url'] );
 		$this->assertArrayNotHasKey( 'query', $parsed_preview_url );
 	}
+
+	/**
+	 * Test filter_user_has_cap.
+	 *
+	 * @covers \CustomizeSnapshots\Customize_Snapshot_Manager::filter_user_has_cap()
+	 */
+	function test_filter_user_has_cap() {
+		$this->mark_incompatible();
+		$post_id = $this->factory()->post->create( array(
+			'post_type' => 'post',
+			'post_status' => 'auto-draft',
+		) );
```

Comment:
On it. For some reason checking for `read_post` cap causes failure in unit tests, haven't figured out why yet, that's why the delay in fixing.